terraform/gcp/modules/compute_engine/variables.tf

```diff
@@ -0,0 +1,16 @@
+
+variable "name"{
+     type=string
```

Comment:
Please add defaults for all variables, "" in this case. Please update the README with the same.

Also, please run terraform fmt --recrusive against the terraform folder.

Reply:
@qburst-praven Formatting is done and changes committed